tools.mk

```diff
@@ -21,3 +21,8 @@ ifeq ($(strip $(diag2cbor)),)
 $(error diag2cbor.rb not found. To install diag2cbor.rb: 'gem install cbor-diag')
 endif
 
+cbor2pretty ?= $(shell command -v cbor2pretty.rb)
+ifeq ($(strip $(cbor2pretty)),)
+$(error cbor2pretty.rb not found. To install cbor2pretty.rb: 'gem install cbor-diag')
```

Comment:
Does gem install cbor-diag also install cbor2pretty.rb?

Reply:
👍

Reply:
(If only GNU make had a nice way to express OR in conditionals, I'd have avoided all the repetition there.)